bin/orca.js

```diff
@@ -2,7 +2,7 @@
 
 const path = require('path')
 const { spawn } = require('child_process')
-const electronPath = require('electron')
+const electronPath = process.env['ELECTRON_PATH'] ? process.env['ELECTRON_PATH'] : require('electron')
```

Comment:
Why do we need this?

Reply:
It is not mandatory for this PR but it provides a way to use a manually installed `electron` in the event that [electron-download](https://github.com/electron-userland/electron-download) fails to install pre-built binary for your platform.

I could remove it but it's a must-have in my current dev environment and maybe it also the case for others. It simply gives more flexibility. What do you think?